docs/user-guide/federation/configmap.md

```diff
@@ -0,0 +1,86 @@
+---
+---
+
+This guide explains how to use ConfigMaps in Federation control plane.
```

Comment:
Missing an article.

"in a Federation control plane" or "with your Federation control plane."